andela/codepirates-ah-frontend
# #167313364 Build out user create article Front-end

#### What does this PR do?
Has create article front-ent page
#### Description of Task to be completed?
- Added functionality for the user to create articles
#### How should this be manually tested?
Download or clone the tutorial project code from this [repo,](https://github.com/andela/codepirates-ah-frontend) in your terminal then git checkout ` ft-article-create-167313364`, install all required NPM packages by running ` npm install ` from the command line in the project root folder (where the package.json is located).

Start the API by running ` npm start ` from the command line in the project root folder, you should see the message Server listening on port 8080 then open your browser 

#### Any background context you want to provide?
- implemented this feature using react 
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167313364
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/16412410/66542148-0964bd80-eb32-11e9-82c9-0d514d628256.png)

![image](https://user-images.githubusercontent.com/16412410/66542155-0f5a9e80-eb32-11e9-9561-86267b51902f.png)

![image](https://user-images.githubusercontent.com/16412410/66542189-2d280380-eb32-11e9-8f71-1142b55dc610.png)

![image](https://user-images.githubusercontent.com/16412410/66995526-74346c80-f0cf-11e9-8ba1-f1661ab928b4.png)

![image](https://user-images.githubusercontent.com/16412410/67008139-2e849d80-f0e9-11e9-9bff-a36aebe7b6f0.png)

#### Questions:
